recipes/astra-toolbox/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "astra-toolbox" %}
+{% set version = "1.9.9.dev5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://anaconda.org/{{ name }}/{{ name }}/{{ version }}/download/osx-64/astra-toolbox-1.9.9.dev5-py38h39e3cac_0.tar.bz2
```

Comment:
I am not sure why the source code is taken from anaconda.org, but not from GitHub. I saw the latest tag on GitHub is https://github.com/astra-toolbox/astra-toolbox/releases/tag/v1.8.3...

Reply:
Looks like this recipe tries to copy the pre-built binaries from the astra-toolbox channel instead of building them against the conda-forge pinnings.

Reply:
Hi @mrakitin and @carterbox, I tried to get this on conda-forge a while ago, but since I didn't really need it, I gave up quite quickly. If you can get it to work that would be nice of course!